### DISCUSSION TITLE
[ente web] Cannot open single images

### DISCUSSION THREAD

**Discussion by roughnecks** [ente-io/ente#1476] [Q&A]
Hello,
continuing the discussion from here: https://github.com/ente-io/ente/issues/1472

I did checkout deploy/photos branch and started over.
When doing `yarn install` I have lots of these warning:

```
warning "workspace-aggregator-f17d6f48-64d3-43ee-ae4f-88d5844f4319 > photos > react-datepicker > react-onclickoutside@6.13.0" has unmet peer dependency "react-dom@^15.5.x || ^16.x || ^17.x || ^18.x".
warning "workspace-aggregator-f17d6f48-64d3-43ee-ae4f-88d5844f4319 > photos > react-datepicker > react-popper@2.3.0" has unmet peer dependency "react@^16.8.0 || ^17 || ^18".
```

I tried installing `node-react` and `node-react-dom` via apt but nothing changed.
Could this be the issue?

**Comment by mnvr** [comment]
No, these warnings are harmless.
